components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.html

```diff
@@ -0,0 +1,132 @@
+            </chef-error>
+          </chef-form-field>
+          <chef-form-field id="update-description">
+            <label>
+              Description <span aria-hidden="true">*</span>
+              <input chefInput formControlName="description" type="text" autocomplete="off"
+                data-cy="update-chefServer-description">
+            </label>
+            <chef-error
+              *ngIf="(updateServerForm.get('description').hasError('required') || updateServerForm.get('description').hasError('pattern')) && updateServerForm.get('description').dirty">
+              Description is required.
+            </chef-error>
+          </chef-form-field>
+          <chef-form-field id="update-fqdn">
+            <label>
+              FQDN <span aria-hidden="true">*</span>
+              <input chefInput formControlName="fqdn" type="text" autocomplete="off"
+                data-cy="update-chefServer-fqdn">
+            </label>
+            <chef-error
+              *ngIf="(updateServerForm.get('fqdn').hasError('required') || updateServerForm.get('fqdn').hasError('pattern')) && updateServerForm.get('fqdn').dirty">
+              FQDN is required.
+            </chef-error>
+          </chef-form-field>
+          <chef-form-field id="update-ip_address">    
+            <label>
+              IP Address <span aria-hidden="true">*</span>
+              <input chefInput formControlName="ip_address" type="text" autocomplete="off"
+                data-cy="update-chefServer-ip_address">
+            </label>
+            <chef-error
+              *ngIf="(updateServerForm.get('ip_address').hasError('required') || updateServerForm.get('ip_address').hasError('pattern')) && updateServerForm.get('ip_address').dirty">
+              IP Address is required.
+            </chef-error>
+          </chef-form-field>  
+          <div id="button-bar">
+            <chef-button [disabled]="isLoading || !updateServerForm.valid || !updateServerForm.dirty"
+              primary inline (click)="saveServer()">                
+            <chef-loading-spinner *ngIf="saving"></chef-loading-spinner>
+            <span *ngIf="saving">Saving...</span>
+            <span *ngIf="!saving">Save</span></chef-button>
+            <span id="saved-note" *ngIf="saveSuccessful && !updateServerForm.dirty">All changes saved.</span>
+          </div>
+        </form>
+      </section>
+      <section class="page-body" *ngIf="tabValue === 'orgs'">
+        <ng-container>
+          <chef-toolbar>
+            <chef-button id="create-button" primary (click)="openCreateModal('create')">Create Org</chef-button>
+          </chef-toolbar>
+          <chef-table-new>
+            <chef-table-header>
+              <chef-table-row>
+                <chef-table-header-cell>Org Name</chef-table-header-cell>
+                <chef-table-header-cell>Admin</chef-table-header-cell>
+                <chef-table-header-cell class="controls"></chef-table-header-cell>
```

Comment:
We recently phased out `controls` in favor of `three-dot-column`. Change here and on L120. (Also update CSS, of course.)
```suggestion
                <chef-table-header-cell class="three-dot-column"></chef-table-header-cell>
```